docs/Installation.rst

```diff
@@ -55,3 +55,19 @@ RPM Based - Fedora 24+ / CentOS 7
    .. code:: bash
 
        $ sudo yum install lago-ovirt
+
+3.  Install ovirt-engine-sdk-python==4.2.7
+
+    pycurl is a dependency required when installing ovirt-engine-sdk-python
+    
+    An error message received:
```

Comment:
You can add a section for troubleshooting like we have in lago: http://lago.readthedocs.io/en/latest/Installation.html